actix-http/CHANGES.md

```diff
@@ -9,6 +9,9 @@
 * `Accept::mime_precedence` and `Accept::mime_preference`. [#1793]
 * `TryFrom<u16>` and `TryFrom<f32>` for `http::header::Quality`. [#1797]
 
+### Changed
+* Bumped `rand` to `0.8
+`
```

Comment:
Place under Unreleased and remove the rogue backtick.

Reply:
Thanks